README.md

```diff
@@ -1 +1,223 @@
-# JavaScript-Capstone-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./src/image/Hummingbird.png" alt="logo" width="200"  height="auto" />
+  <br/>
+
+  <h3><b>JavaScript Capstone Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Video Description](#video-description)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 JavaScript Capstone Project <a name="about-project"></a>
+
+**JavaScript Capstone Project** is a website to share informations about episodes of a TV show.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/concepts/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **TVmaze API**
+- **Linters**
+- **Dynamic Section**
+- **Webpack**
+- **Jest Unit Testing**
+- **Involvement API**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://al-amin-khan-shakil.github.io/JavaScript-Capstone-Project/dist/)
+
+<!-- Video Description -->
+
+## Video Description <a name="video-description"></a>
+
+
+- [Description Link](https://drive.google.com/file/d/1u3QZl2o-Fir3uJxmQuL5i_4OCSpy_orX/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] This presentation was engaging and descriptive enough I must say 👏 👏, but in accordance to the requirement; kindly specifically highlight the piece of code or  something you built that you are very proud of. 👍 👍

<img width="642" alt="Untitled" src="https://github.com/Al-Amin-Khan-Shakil/JavaScript-Capstone-Project/assets/103386183/c2292418-23ec-4714-86e9-d254c90261d1">

Reply:
Sorry, I didn't get your point. Can you please describe it elaborately? Because me and my partner both highlighted 2 pieces of code in our video I highlighted the displayList() function and my partner highlighted commentData() and fetchcomment() function.

Reply:
@Al-Amin-Khan-Shakil and @AlselwiAisha I believe you're cleared with the explanation I gave during the video call, the pieces of code you talked about was on a general note. You did not mention any thing as regards the **''interesting''** or **''proud of''** part as required in the project. 👍